lib/modlab.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+exports.greet = function(name) {
+  console.log(name);
+  return 'hello ' + name;
+};
```

Comment:
To suppress your node modules from github, add a .gitignore file along with your package.json in the next assignment. The .gitignore file should contain the text 'node_modules/'